peachjam/views/accounts.py

```diff
@@ -19,3 +21,19 @@ def get_success_url(self):
 
     def get_object(self, queryset=None):
         return self.request.user
+
+
+class GetAccountView(View):
+    def get_object(self):
+        return self.request.user
+
+    def get(self, *args, **kwargs):
+        if self.request.user.is_authenticated:
+            user_details = {
+                "id": self.request.user.id,
+                "email": self.request.user.email,
+                "username": self.request.user.username,
```

Comment:
The username isn't useful - rather let's add `name` which is first_name + last_name.

Reply:
Do all users have first and last names? I remember being able to log in with only a username